ovirt/resource_ovirt_datacenter.go

```diff
@@ -80,7 +83,7 @@ func resourceOvirtDataCenterUpdate(d *schema.ResourceData, meta interface{}) err
 			datacenterBuilder.Name(name.(string))
 		}
 	} else {
-		return fmt.Errorf("DataCenter's name does not exist!")
+		return fmt.Errorf("datacenter's name does not exist")
```

Comment:
Although the capitalized starting word in error string is unidiomatic, the terraform official providers all do so, which I think is a kind of following the rule of terraform log `ovirt_datacenter.my_dc: Importing from ID "68e4fc08-4815-408f-a03a-4ea187aee1a5"`. Please refer to [AWS error string](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_instance.go#L572).

Reply:
regarding capitalized words in error format is my mistake that should not go into this PR, sorry will revert back.